CONTRIBUTING.md

```diff
@@ -56,6 +56,31 @@ release branch. For example, if the most recent branch is `release-0.2`, and the
 development for v0.3.0, a bug fix that merged to `master` that also affects `v0.2.x` may be considered for backporting
 to `release-0.2`. We generally do not accept PRs against older release branches.
 
+### Features and bugs
+
+Open [issues](https://github.com/kubernetes-sigs/cluster-api/issues/new/choose) to report bugs, or minor features.
+
+For big feature, API and contract amendments, we follow the CAEP process as outlined below.
+
+### Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
+
+- The template, and accepted proposals live under `docs/proposals`.
+- A proposal SHOULD be introduced and discussed during the weekly community meetings.
```

Comment:
Since not all community members can attend the weekly meetings due to time zone issues, we should attempt to be a bit more inclusive here:
```suggestion
- A proposal SHOULD be introduced and discussed during the weekly community meetings, [Kubernetes SIG Cluster Lifecycle mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle), or [discuss forum](https://discuss.kubernetes.io/c/contributors/cluster-api/).
```